libs/net/entities/CBRAReportStaffSummary.cs

```diff
@@ -13,5 +13,5 @@ public class CBRAReportStaffSummary
     public string Staff { get; set; } = "";
 
     [Column("cbra_hours")]
-    public decimal CbraHours { get; set; }
+    public string CbraHours { get; set; }
```

Comment:
Are you changing the type because it is actually a string value, or are you changing it because Excel expects a string value.  If the second, then don't change the type.

Reply:
the type is changed in the function, that it returns string not numbers.